Gemfile

```diff
@@ -3,7 +3,7 @@ raise "Ruby versions less than 2.2.2 are unsupported!" if RUBY_VERSION < "2.2.2"
 # VMDB specific gems
 #
 
-gem "rails",                           "~>5.0.0.1"
+gem "rails",                           "~>5.0.0", ">= 5.0.0.1"
```

Comment:
Yeah this allows 5.0.1+ does it not? We had agreed that Darga and Euwe should be restricted to 5.0.0 due to incompatibilities with virtual_attributes that we don't want to bother backporting.